scripts/provisioning/s3setup/s3setup/cortx_s3_setup.py

```diff
@@ -26,6 +26,7 @@
 from s3backgrounddelete.cortx_cluster_config import CipherInvalidToken
 from cortx.utils.validator.v_pkg import PkgV
 from cortx.utils.validator.v_service import ServiceV
+from cortx.utils.validator.v_path import PathV
```

Comment:
I guess we are not following, what we followed for confstore, cipher and message-bus utilities of py-utils, that we need to write wrapper for all py-utils packages we use.

Sorry I did not catch this in your earlier PR.

I guess we need to make a new wrapper: "s3validator", just like "s3confstore" or "s3cipher",  and then you import that wrapper package here and use it.

Reply:
that way the validator can be called directly from s3_setup script.

Reply:
I was not aware about this wrapper approach while using modules from py-utils until now. Also are not we going to replace shell version s3_setup soon with python version s3setup? If so, then what will be the advantage of this wrapper approach in near future?

Reply:
Also I think it is better to use a separate ticket to write those wrappers.

Reply:
I think we can live without the wrapper. I can not think of any very good reason why the wrapper will help, it may turn out more effort than benefit. Lets discuss if there are any obvious benefits.

Reply:
As discussed, lets not make wrapper for py-utils: validator, just use it here.